bot/exts/discord/discord.py

```diff
@@ -49,7 +49,7 @@ async def embed(
         author: str = None,
         author_icon_url: str = None,
     ) -> None:
-        """Preview a discrod embed."""
+        """Preview a discord embed."""
```

Comment:
```suggestion
        """Preview a Discord embed."""
```
Let's make this capitalized, since Discord is a company name.